engines/cine/gfx.cpp

```diff
@@ -93,6 +93,377 @@ static const byte mouseCursorCross[] = {
+	"",									// DOS only (no DOS version exists)
+	"Future Wars: Time Travellers",		// Amiga/Atari only
+	""									// DOS only (no DOS version exists)
+};
+
+static const char *endCreditsFW[] = {
+	// English, first line
+	"Design: Paul Cuisset",
+	// English, second line DOS
+	"IBM PC Version: Daniel Morais",
+	// English, second line Amiga/Atari
+	"Program: Paul Cuisset",
+	// English, remainder of credits
+	"Graphics: Eric Chahi\n"
+	"Music: Jean Baudlot\n"
+	"Sound effects: Antoine O'Heix\n"
+	"Delphine Studio Midi: Marc Minier\n"
+	"Technical help: Philippe Chastel, Jesus Martinez\n"
+	"Digitized sounds sampling: Philippe Chastel, Paul Cuisset, Antoine O'Heix, Jesus Martinez\n"
+	"Cinematique interpreter: Paul Cuisset\n"
+	"Many thanks to: Philippe Delamarre, Emmanuel Lecoz, Michael Sportouch, Peter Stone, Matthew Tims, "
+	"Patricia Vermander",
+
+	// French, first line
+	"Scenario: Paul Cuisset",
+	// French, second line DOS
+	"Version IBM PC: Daniel Morais",
+	// French, second line Amiga/Atari
+	"Programmation: Paul Cuisset",
+	// French, remainder of credits
+	"Graphismes: Eric Chahi\n"
+	"Musiques: Jean Baudlot\n"
+	"Bruitages: Antoine O'Heix\n"
+	"Delphine Studio Midi: Marc Minier\n"
+	"Aide technique: Philippe Chastel, Jesus Martinez\n"
+	"Echantillonage sons numeriques: Philippe Chastel, Paul Cuisset, Antoine O'Heix, Jesus Martinez\n"
+	"Systeme Cinematique: Paul Cuisset\n"
+	"Nous tenons \205 remercier: Philippe Delamarre, Emmanuel Lecoz, Michael Sportouch, Peter Stone, Matthew Tims, "
+	"Patricia Vermander",
+
+	// German, first line
+	"Entwurf: Paul Cuisset",
+	// German, second line DOS
+	"IBM PC Version: Daniel Morais",
+	// German, second line Amiga/Atari
+	"Programmierung: Paul Cuisset",
+	// German, remainder of credits
+	"Grafiken: Eric Chahi\n"
+	"Musiken: Jean Baudlot\n"
+	"Soundeffekte: Antoine O'Heix\n"
+	"Delphine Studio Midi: Marc Minier\n"
+	"Technische Hilfe: Philippe Chastel, Jesus Martinez\n"
+	"Sampling digitalisierter Kl\204nge: Philippe Chastel, Paul Cuisset, Antoine O'Heix, Jesus Martinez\n"
+	"Cinematique System: Paul Cuisset\n"
+	"Wir danken: Philippe Delamarre, Emmanuel Lecoz, Michael Sportouch, Peter Stone, Matthew Tims, "
+	"Patricia Vermander",
+
+	// Spanish, first line
+	"Dise\244o: Paul Cuisset",
+	// Spanish, second line DOS
+	"Versi\242n de IBM PC: Daniel Morais",
+	// Spanish, second line Amiga/Atari
+	"Programaci\242n: Paul Cuisset",
+	// Spanish, remainder of credits
+	"Gr\240ficos: Eric Chahi\n"
+	"M\243sicas: Jean Baudlot\n"
+	"Efectos de sonido: Antoine O'Heix\n"
+	"Delphine Studio Midi: Marc Minier\n"
+	"Ayuda Tecnica: Philippe Chastel, Jesus Martinez\n"
+	"Muestreo de sonidos digitalizados: Philippe Chastel, Paul Cuisset, Antoine O'Heix, Jesus Martinez\n"
+	"Sistema Cinematique: Paul Cuisset\n"
+	"Muchas gracias a: Philippe Delamarre, Emmanuel Lecoz, Michael Sportouch, Peter Stone, Matthew Tims, "
+	"Patricia Vermander",
+
+	// Italian, first line
+	"Disegno: Paul Cuisset",
```

Comment:
"Disegno" is wrong. As in the opening credits you can choose between Idea/Ideazione/Progettazione/Design